lib/mavenlink/workspace_allocation.rb

```diff
@@ -1,4 +1,7 @@
 module Mavenlink
   class WorkspaceAllocation < Model
+    def split_allocation(date)
+      client.put("#{collection_name}/split", split_date: date, workspace_allocation_id: id)
```

Comment:
Since `/split` will modify the existing allocation you call it on and return the newly created one I think it would be good to:

1. Update the calling allocation's attributes
2. Return an instance of the newly created allocation

Reply:
Gotcha, fixed now.